Gems/Atom/RHI/Vulkan/Code/Source/RHI/Framebuffer.cpp

```diff
@@ -79,7 +79,8 @@ namespace AZ
 
         const RHI::Size& Framebuffer::GetSize() const
         {
-            return m_attachments.front()->GetImage().GetDescriptor().m_size;
+            static const RHI::Size safeSize;
+            return m_attachments.size() ? m_attachments.front()->GetImage().GetDescriptor().m_size : safeSize;
```

Comment:
Can we add an assert or a warning wherever this is called and we expect is to be not null just so we now not silently passing that case?

Reply:
I will add an assert later when I work out how to fix the zero size array from happening. For now this is just to prevent people crashing.

Reply:
any warning here will end up really chatty. if anything, can we warnonce?